internal/myks/globe.go

```diff
@@ -85,6 +89,9 @@ type Globe struct {
 	// Git repository URL
 	GitRepoUrl string
 
+	// Prefix for kubernetes namespacesM only used in helm rendering
```

Comment:
```suggestion
	// Prefix for kubernetes namespaces, only used in helm rendering
```

Side note: why do we even need that? 🤔